src/components/SimpleCommonComponents/Tooltip/index.tsx

```diff
@@ -45,6 +48,14 @@ const Tooltip = ({
       arrow({
         element: arrowRef,
       }),
+      size({
```

Comment:
@matthiasblum this is how I'm trying to solve the issue about the overgrown tooltip. It seems to work well here and in other cases , but let me know if you notice the tooltip misbehaving somewhere else